pkg/api/converterfromapi.go

```diff
@@ -549,6 +549,18 @@ func convertKubernetesConfigToVLabs(api *KubernetesConfig, vlabs *vlabs.Kubernet
 	vlabs.ClusterSubnet = api.ClusterSubnet
 	vlabs.NetworkPolicy = api.NetworkPolicy
 	vlabs.DockerBridgeSubnet = api.DockerBridgeSubnet
+	vlabs.NodeStatusUpdateFrequency = api.NodeStatusUpdateFrequency
```

Comment:
Good question. Empty fields get their version-specific defaults here in the generate flow:

https://github.com/Azure/acs-engine/pull/964/files#diff-8edb2653431a5da9ad1f4286097702a7R97